service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/ConditionLinkSetMapper.java

```diff
@@ -72,6 +74,13 @@ public String mapConditionToLinkSet(Condition condition, boolean isNested) {
 
         builder.code(buildCode(condition));
 
+        if (condition.hasAsserter()) {
+            var asserterReference = condition.getAsserter();
```

Comment:
it would be good to check here is the asserter is referencing Practitioner not other resource.

Reply:
Could you please also add test to not map the asserter if it is not practitioner ?

Reply:
only practitioner is profiled, so this would be unlikely to occur